modules/DNN/NeuronLayer.cpp

```diff
@@ -4,36 +4,50 @@
 #include "macroses.h"
 
 namespace DirectGraphicalModels { namespace dnn
-{
-	void CNeuronLayer::generateRandomWeights(void)
 	{
-		m_weights = random::U(m_weights.size(), m_weights.type(), -0.5f, 0.5f);
-		m_biases = random::U(m_biases.size(), m_biases.type(), -0.5f, 0.5f);
-	}
+		void CNeuronLayer::generateRandomWeights(void)
```

Comment:
Please do not change the methods which are not affected by the Pull Request. Changing just indentation makes it extremely difficult to track other important changes in the code